xilem_web/src/app.rs

```diff
@@ -1,44 +1,48 @@
 // Copyright 2023 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::{DomNode, ViewCtx};
 use std::{cell::RefCell, rc::Rc};
 
-use crate::{
-    context::Cx,
-    view::{DomNode, View},
-    Message,
-};
-use xilem_core::{Id, MessageResult};
+use crate::{DomView, DynMessage, PodMut};
+use xilem_core::{MessageResult, ViewId};
+
+type IdPath = Vec<ViewId>;
```

Comment:
Have you considered `Arc<[ViewId]>` for this?

Reply:
I haven't but it definitely makes sense and was actually a simple change (with `Rc` though, because `Arc` isn't necessary (yet)). Thanks for the hint.